### DISCUSSION TITLE
Welcome to flow Discussions!

### DISCUSSION THREAD

**Discussion by pacexy** [pacexy/flow#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by monsoon7ice** [comment]
这个阅读器很好用。
如果可以，请添加**复制文字**的功能，谢谢！
